### PR TITLE
[LF] [69930176] Get specs passing

### PR DESCRIPTION
**Pull request by lfender6445**
Ignore the story number in the branch name. Should be 000
- Add .bowerrc, match AG setup, edit gitignore
- JS specs are now running 
- Cleanup

Here is a similar PR https://github.com/primedia/zutron.js/pull/6/files

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -1,4 +0,0 @@
-Changelog
```
Comment (c0) [inline Changelog.md:1]: Curious – why was this removed?

File: README.md [hunk 2]
```diff
@@ -7,6 +7,7 @@ ___________________________
 To run jasmine tests:
 
 1. `bundle install`
-1. `bower install`
-1. `rake jasmine`
+2. `bower install`
+3. `rake jasmine`
+4. `./watch.sh`
```
Comment (c0) [inline README.md]: What do you think about switching this to a Procfile to match zutron.js and the other libs?

Comment (lfender6445) [inline README.md]: I like the idea. Foreman makes this much easier. I'll push up a change.

File: Changelog.md (hunk 1 above)
Comment (lfender6445) [inline Changelog.md:1]: I figured we're all bright enough to run `git log -- bower.json` and grep for jquery - it's also one of the only packages that has a changelog and I don't see our devs using them in other apps. Do you see strong benefit to keeping it?

Comment (c0) [inline Changelog.md:1]: Yeah, it's one of those things where you either keep it up to date or remove it. I'm good with removing it since we use the git history extensively.

**Comment by c0** [comment]
:+1:
